experiments/run_slicegpt.py

```diff
@@ -8,9 +8,13 @@
 import shutil
 
 import torch
-import wandb
+from transformers.models.llama.modeling_llama import LlamaConfig
+from transformers.models.phi.modeling_phi import PhiConfig
 
+import wandb
 from slicegpt import data_utils, gpu_utils, hf_utils, layernorm_fusion, rotate, utils
+from slicegpt.adapters.hf_compatible_phi import SlicedPhiForCausalLM
```

Comment:
`SlicedPhiForCausalLM` is in `sliced_phi.py`. Looks like this file was renamed and the import is using the old name.